.npmignore

```diff
@@ -0,0 +1,4 @@
+*swp
```

Comment:
oh - thanks for thinking of this!

I just did a search around and turns out you don't need this cause 

> If there's no .npmignore file, but there is a .gitignore file, then npm will ignore the stuff matched by the .gitignore file.

(from https://docs.npmjs.com/misc/developers)